src/Avalonia.Base/Input/MouseDevice.cs

```diff
@@ -104,6 +104,9 @@ private void ProcessRawEvent(RawPointerEventArgs e)
                 case RawPointerEventType.Swipe:
                     e.Handled = GestureSwipe(mouse, e.Timestamp, e.Root, e.Position, props, ((RawPointerGestureEventArgs)e).Delta, keyModifiers, e.InputHitTestResult.firstEnabledAncestor);
                     break;
+                case RawPointerEventType.CancelCapture:
+                    PlatformCaptureLost();
```

Comment:
There is a loop here here which is probably unexpected.

Suppose the capture is released naturally (e.g. mouse up on any control), The call chain looks like this:
- `WindowsMousePointer.Capture(null)` ←
- `WindowsMousePointer.PlatformCapture(null)`
- `User32!ReleaseCapture()`
- `WndProc(WM_CAPTURECHANGED)`
- `WindowsMouseDevice.PlatformCaptureLost()`
- `WindowsMousePointer.Capture(null)` ←

While the loop will stop at some point because there's nothing no capture, we should avoid calling `PlatformCaptureLost` if the pointer currently doesn't have the capture – after all, the capture hasn't been lost in this case.